internal/network/machine_network_cidr.go

```diff
@@ -195,6 +195,36 @@ func GetMachineCIDRHosts(log logrus.FieldLogger, cluster *common.Cluster) ([]*mo
 	return ret, nil
 }
 
+func GetClusterNetworks(cluster *common.Cluster, log logrus.FieldLogger) []string {
+	var err error
+	cidrs := make(map[string]bool)
```

Comment:
Why an hashmap? Make it a list and append to it

Reply:
For uniqueness. Actually I needed a set, but go doesn't have sets.

Reply:
@YuviGold This is a Golang patterns for sets: https://www.davidkaya.com/sets-in-golang/